apps/meteor/ee/server/api/ldap.ts

```diff
@@ -24,7 +24,7 @@ API.v1.addRoute(
 				throw new Error('LDAP_disabled');
 			}
 
-			await LDAPEE.sync();
+			await Promise.all([LDAPEE.sync(), LDAPEE.syncAvatars()]);
```

Comment:
```suggestion
			await LDAPEE.sync();
			await LDAPEE.syncAvatars();
```

Running them at the same time would cause some race conditions and keep multiple connections active. It would also not sync the avatars of new users until the next full sync.

Reply:
updated!